### PR TITLE
Implement better design for social wall messages and OpenGraph markup

### PR DESCRIPTION
**Pull request by jloguercio**
Okey, this makes a better design to the Social Wall for the shared profile,  this changes shows a separator panel wall post by wall post, to differentiate between Post, also improve the markup for the opengraph functionality.

I failed to get the opengraph typing the url because is a mess trying to identify the url for example if i type it following by a text or something like that (still trying to find a better solution)

To see the changes in opengraph functionality you need to add a new link to your wall

### CODE REVIEW

File: main/inc/lib/social.lib.php [hunk 1]
```diff
@@ -1578,28 +1579,24 @@ private static function headerMessagePost($authorId, $receiverId, $users, $messa
      */
     public static function readContentWithOpenGraph($link)
     {
+        if(strpos($link,"://")===false && substr($link,0,1)!="/") $link = "http://".$link;
```
Comment (ywarnier) [inline main/inc/lib/social.lib.php]: Missing {} and space after "if"

**Comment by jloguercio** [comment]
Update : remove (for now) popularity post vote system, I think it can safety merge.

File: main/inc/ajax/social.ajax.php [hunk 2]
```diff
@@ -225,14 +225,17 @@
         // Read the Url using OpenGraph and returns the hyperlinks content
     case 'readUrlWithOpenGraph':
         $url = isset($_POST['social_wall_new_msg_main']) ? $_POST['social_wall_new_msg_main'] : '';
+        $url = trim($url);
```
Comment (ywarnier) [inline main/inc/ajax/social.ajax.php]: Y de donde viene $url?

Comment (jloguercio) [inline main/inc/ajax/social.ajax.php]: uhmm de la misma linea de arriba $url, solo le hago un trim a esa misma linea y la reasigno a la misma variable

**Comment by ywarnier** [comment]
Probé el cambio en local pero no veo diferencia en la interfaz... que se supone que cambie?

**Comment by jloguercio** [comment]
Primer hacer un conposer install

**Comment by jloguercio** [comment]
Luego ingresar un link y bajo ese link ingresar comentarios de otros usuarios

**Comment by jloguercio** [comment]
He dejado también los espacios necesarios para terminar lo del popularity vote

**Pull request merged**